recipes/pyviz/meta.yaml

```
@@ -0,0 +1,70 @@
+{% set name = "pyviz" %}
+{% set version = "0.10.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pyviz/{{ name }}/archive/{{ version }}.tar.gz
```

```suggestion
  url: https://pypi.io/packages/source/p/{{name}}/{{name}}-{{ version }}.tar.gz
```

Need to get the `.version` file correctly. Consistent with the sources used for other pyviz packages. SHA update below...